src/js/index.js

```diff
@@ -101,11 +92,54 @@ export function getElementsBondsData(element1, element2, order = undefined) {
  * @returns {Number}
  */
 export function getElementAtomicRadius(elementSymbol) {
-    const elementName = Object.keys(PERIODIC_TABLE).find(
-        (key) => elementSymbol === PERIODIC_TABLE[key].symbol,
-    );
-    if (elementName) {
-        return PERIODIC_TABLE[elementName].atomic_radius_pm * 0.01;
+    const config = PERIODIC_TABLE[elementSymbol];
+    if (config) {
+        return PERIODIC_TABLE[elementSymbol].atomic_radius_pm / 100;
     }
     return 1.0;
 }
+
+/**
+ *
+ * @param elements {String[]} - List of elements (as symbols)
+ * @param properties {String[]} - List of properties
+ * @param propertiesMap {Object} - Object mapping property names to custom names (e.g. {'atomic_radius_pm': 'radius'})
+ * @param separator {String} - Separator string for the suffix (default: ':')
+ * @returns {Object[]} - List of Objects {property_key: property_value}
+ * @example
+ * getAtomicPropertiesFlat(
+ *     ["H", "Na"],
+ *     ["atomic_radius_pm", "atomic_number", "pauling_negativity"],
+ *     { atomic_radius_pm: "atomic_radius" }
+ * );
+ * // returns [
+ * //  { 'atomic_radius:H': 25 },
+ * //  { 'atomic_radius:Na': 180 },
+ * //  { 'atomic_number:H': 1 },
+ * //  { 'atomic_number:Na': 11 },
+ * //  { 'pauling_negativity:H': 2.2 },
+ * //  { 'pauling_negativity:Na': 0.93 },
+ * // ]
+ */
```

Comment:
We can remove `//` in the example

Reply:
Unnecessary comments have been removed and on my end, it still seems to render correctly.